src/content/documentation/develop/best-practices-for-collecting-user-data-consents.md

```diff
@@ -148,16 +148,14 @@ Before designing your consent dialogs, you should determine the privacy consent
 In this example:
 
 1. Your extension should start up with all data collection functionality disabled.
-2. Shortly after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
+2. Immediately after installation (see [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/)), present them with a consent dialog.
```

Comment:
The current construction of this sentence splits the concept being conveyed with a parenthetical. This makes the sentence more difficult for a reader to follow than necessary.

```suggestion
2. Immediately after installation, present them with a consent dialog. (See [Best practices for onboarding, upboarding, and offboarding users](/documentation/develop/onboard-upboard-offboard-users/) for more.)
```